runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/metrics/MetricContext.java

```diff
@@ -25,6 +27,11 @@
 
     Metric.Kind kind();
 
+    default Metric.StreamDirection streamDirection()
+    {
+        return BOTH;
+    }
+
```

Comment:
Do we need this on both `Metric` and `MetricContext`?

Reply:
No, we actually need it only in `MetricContext`. It made sense to add it to `Metric` semantically speaking as it feels like a core property of a metric (like `kind` or `unit`), but I can remove it from there and it would still work. Pls let me know if I should go ahead and remove it.

Reply:
Yes, please remove it from `Metric`.

Actually, this feels more like a property of the metric handler, so `MetricContext` is a better place for it.

Reply:
done